test/unit/model/Model.spec.ts

```diff
@@ -8,10 +8,6 @@ describe('unit/model/Model', () => {
     @Attr() id!: number
   }
 
-  it('throws when accessing the database but it is not injected', () => {
-    expect(() => new User().$database()).toThrow()
-  })
```

Comment:
The database is no longer injected into the model so we no longer need this test.